automation/addLabelsOnPullRequest.automation.js

```diff
@@ -102,63 +101,63 @@ exports.pullRequestWIPLabelAutomation = async context => {
 
 	const issueLabels = await listIssueLabels(context);
 	const foundWIPLabel = issueLabels.data.filter(
-		label => label.name === ':construction: WIP'
+		label => label.name === ":construction: WIP"
 	);
 	const foundReadyForReviewLabel = issueLabels.data.filter(
-		label => label.name === ':mag: Ready for Review'
+		label => label.name === ":mag: Ready for Review"
 	);
 
 	if (
-		(title.includes('WIP') ||
-			title.includes('Work In Progress') ||
-			title.includes('work in progress') ||
-			title.includes(':construction:')) &&
+		(title.includes("WIP") ||
```

Comment:
Codacy has a fix for the issue: [Replace `↹↹` with `····`](https://app.codacy.com/gh/aayushchugh/bot/pullRequest?prid=10060881)
```suggestion
    (title.includes("WIP") ||
```